pontoon/base/static/js/fluent_interface.js

```diff
@@ -57,43 +57,55 @@ var Pontoon = (function (my) {
         }
         // Plurals
         else if (self.isPlural(entityAST)) {
+          function renderPluralEditor(value, name, pluralIndex) {
+            var example = Pontoon.locale.examples[pluralIndex];
+
+            $('#ftl-area .main-value ul')
+              .append(
+                '<li class="clearfix">' +
+                  '<label class="id built-in" for="ftl-id-' + name + '">' +
+                    '<span>' + name + '</span>' +
+                    (self.isCLDRplural(name) && example ? '<span> (e.g. ' +
+                      '<span class="stress">' + example + '</span>' +
+                    ')</span>' : '') +
+                    '<sub class="fa fa-remove remove" title="Remove"></sub>' +
+                  '</label>' +
+                  self.getTextareaElement(name, value) +
+                '</li>'
+              );
+          }
+
           var pluralForms = Pontoon.locale.cldr_plurals.map(function (item) {
             return Pontoon.CLDR_PLURALS[item];
           });
 
-          var variants;
-          if (translationAST) {
-            variants = translationAST.value.elements[0].variants;
-          }
-
           if (!Pontoon.locale.examples) {
             Pontoon.generateLocalePluralExamples();
           }
 
-          pluralForms.forEach(function (pluralStr, pluralInt) {
-            var value = '';
+          // Translated
+          if (translationAST) {
+            var variants = translationAST.value.elements[0].variants;
 
-            if (translationAST) {
-              for (var i = 0; i < variants.length; i++) {
-                if (variants[i].key.name === pluralStr) {
-                  value = self.serializePlaceables(variants[i].value.elements);
-                  break;
-                }
-              }
+            for (var i = 0; i < variants.length; i++) {
```

Comment:
This should be a `forEach` instead. The reason why it wasn't before is because there was a `break` in there, but you've removed it.